copy-fixtures.js

```diff
@@ -0,0 +1,31 @@
+import { promisify } from "node:util";
+import chip from "node:child_process";
+import fs from "node:fs";
+import path from "node:path";
+
+const exec = promisify( chip.exec );
+
+await exec( "npm install --no-save jquery" );
+
+const fixtures = path.join( ".", "test", "fixtures" );
+
+try {
+	await fs.promises.mkdir( path.join( ".", "test" ) );
```

Comment:
`fs.promises` are discouraged; instead, use:
```js
import fs from  "node:fs/promises";
```

Reply:
Thank you for this information. You learn something new every day.